modules/monitoring-silencing-alerts.adoc

```diff
@@ -10,7 +10,10 @@ You can either silence a specific alert or silence alerts that match a specifica
 
 .Prerequisites
 
-* You have access to the cluster as a developer or as a user with `edit` permissions for the project that you are viewing metrics for.
+* You are a cluster administrator and have access to the cluster as a user with the `cluster-admin` role.
+* You are a non-administator user and have access to the cluster as a user with the following user roles:
+** The `cluster-monitoring-view` role, which allows you to access Alertmanager.
+** The `monitoring-alertmanager-edit` role, which permits you to create and silence alerts.
```

Comment:
`cluster-monitoring-view` and `monitoring-alertmanager-edit` are only required to silence alerts from the Admin console. For the Dev console, I believe that the `monitoring-rules-edit` role is required.

![image](https://user-images.githubusercontent.com/2587585/206407537-0b15b5a9-de12-4bb7-8f4b-e5066a28fd55.png)

Reply:
and the above steps to silence alert on developer console is fine, checked on 4.9 - 4.12, see from https://pasteboard.co/9r5i7jKThSdI.png, there is a toggle button under `Notifications`, click it and select the silence duration also could silence alert

Reply:
I added the following to the **Prerequisites** section:

`** The `monitoring-rules-edit` role, which permits you to create and silence alerts in the *Developer* perspective in the web console.`